cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -14,6 +14,11 @@
 {% endblock %}
 
 {% block hero %}
+<script type="text/javascript">
+    var string = "{{ api_json}}";
+    console.log( string );
+</script>
+
 <section class="m-hero">
```

Comment:
Would now be a good time to remove the `hero` section, since it is never shown anyway?

Reply:
We'll do it in https://github.com/cfpb/cfgov-refresh/pull/4638